exec/src/extends.ml

```diff
@@ -0,0 +1,27 @@
+(* FreeSpec
+ * Copyright (C) 2018–2019 ANSSI
+ *
+ * Contributors:
+ * 2019 Thomas Letan <thomas.letan@ssi.gouv.fr>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *)
+
+open Interfaces
+
+let register_interface path semantics =
```

Comment:
Having two functions with almost the same name is confusing to me.

Reply:
I guess you are right. I will try to find a better name for one of these functions.